source/libraries/HelperFunctions.ts

```diff
@@ -1,9 +1,12 @@
 import * as binascii from "binascii";
 import * as EthjsAccount from "ethjs-account";
+import * as EthjsQuery from 'ethjs-query';
 
-const DEFAULT_TEST_ACCOUNT_BALANCE = 1 * 10 ** 20; // Denominated in wei
+const DEFAULT_ETHEREUM_BLOCK_INTERVAL_MILLISECONDS: number = 1200;
+const ETHEREUM_BLOCK_INTERVAL_MILLISECONDS: number = process.env.ETHEREUM_BLOCK_INTERVAL_MILLSECONDS ? parseInt(process.env.ETHEREUM_BLOCK_INTERVAL_MILLISECONDS!, 10) : DEFAULT_ETHEREUM_BLOCK_INTERVAL_MILLISECONDS;
```

Comment:
Style: No need for the type on either of these (or the ones below).  It is inferred from the right side of the equal sign.

Reply:
Ah, good call.